MitoMathHelper.js

```diff
@@ -172,4 +172,94 @@ const MitoMathHelper = class MitoMathHelper {
         let normalizedDistance = aToPDoToAToB / aToBSquared;
         return [linePointA[0] + (aToB[0] * normalizedDistance), linePointA[1] + (aToB[1] * normalizedDistance)]
     }
+
+    /**
+     * returns magnitude of a motion
+     * @param vector
+     * @returns {number}
+     * @private
+     */
+    static _getMagnitude(vector) {
+        return Math.hypot(vector[0], vector[1]);
+    }
+
+    /**
+     * returns magnitude squared
+     * @param vector
+     * @returns {number}
+     * @private
+     */
+    static _getMagnitudeSquared(vector) {
+        return vector[0] * vector[0] + vector[1] * vector[1];
+    }
+
+    /**
+     * returns dot product
+     * @param vectorA
+     * @param vectorB
+     * @returns {number}
+     * @private
+     */
+    static _dotProduct(vectorA, vectorB) {
+        return vectorA[0] * vectorB[0] + vectorA[1] * vectorB[1];
+    }
+
+    /**
+     * returns angle in radians between two bodies
+     * @param bodyA
+     * @param bodyB
+     * @returns {number}
+     * @private
+     */
+    static _getAngleBetweenBodies(bodyA, bodyB) {
```

Comment:
you should probably already have both positions from within whatever scope you're calling this function, so just do:

```
Math.atan2(potitionB[1] - positionA[1], positionB[0] - positionA[0]);
```

inside the code.